lib/classes/Variables.js

```diff
@@ -145,7 +148,7 @@ class Variables {
   }
 
   getValueFromSelf(variableString) {
-    let valueToPopulate = _.cloneDeep(this.service);
+    let valueToPopulate = this.service;
```

Comment:
we clone just in case because objects are passed by reference, so any changes to `valueToPopulate` would actually affect the original service object. Is there any reason you removed cloning?